README.md

```diff
@@ -1,9 +1,18 @@
-# Aave 3.0.1 upgrade proposal
+# Aave 3.0.2 upgrade proposal
+
+## Context
+
+The [v3.0.1](https://github.com/aave/aave-v3-core/tree/feat/3.0.1) upgrade introduces a new `flashloanable` configuration boolean per asset, as this value resides on a previously unused storage space, if will default to `false` effectively disabling flashloans. Therefore to create the proposal in a non-breaking manner, it will also set `flashLoanable` to `true` for all assets.
```

Comment:
```suggestion
The [v3.0.1](https://github.com/aave/aave-v3-core/tree/feat/3.0.1) upgrade introduces a new `flashloanable` configuration boolean per asset, as this value resides on a previously unused storage space, it will default to `false` effectively disabling flashloans. Therefore to create the proposal in a non-breaking manner, it will also set `flashLoanable` to `true` for all assets.
```